src/scvi/model/_multivi.py

```diff
@@ -588,7 +589,9 @@ def get_accessibility_estimates(
             return pd.DataFrame(
                 imputed,
                 index=adata.obs_names[indices],
-                columns=adata.var_names[self.n_genes :][region_mask],
+                columns=adata["rna"].var_names[self.n_genes :][region_mask]
+                if type(adata).__name__ == "MuData"
```

Comment:
Is this the check for MuData that we use throughout the code in e.g. totalVI. We should wrap it in a helper function then.

Reply:
fixed